docs/INSTALL.md

```diff
@@ -174,7 +174,20 @@ Otherwise, replace `--bitcoin.testnet` with `--bitcoin.simnet`. If you are
 installing `lnd` in preparation for the
 [tutorial](http://dev.lightning.community/tutorial), you may skip this step.
 ```
-lnd --bitcoin.active --bitcoin.testnet --debuglevel=debug --bitcoin.rpcuser=kek --bitcoin.rpcpass=kek --externalip=X.X.X.X
+lnd --bitcoin.active --bitcoin.testnet --debuglevel=debug --btcd.rpcuser=kek --btcd.rpcpass=kek --externalip=X.X.X.X
+```
+
+#### Running lnd using the bitcoind backend
+
+In order to run `lnd` with a `bitcoind` back-end, the `bitcoind` instance must
+be configured with `--txindex` just like `btcd` above. In addition, you'll need
+ to configure the `bitcoind` instance with `--zmqpubrawblock` and `--zmqpubrawtx`
+(the latter is optional but allows you to see unconfirmed transactions in your
+wallet). They must be combined in the same ZMQ socket address. Then, run this
+command after `bitcoind` has finished syncing on testnet. Otherwise, replace
+`--bitcoin.testnet` with `--bitcoin.regtest`.
+```
+lnd --bitcoin.active --bitcoin.testnet --debuglevel=debug --bitcoin.node=bitcoind --bitcoind.rpcuser=kek bitcoind.rpcpass=kek --externalip=X.X.X.X
```

Comment:
We may want to note that you don't need to set the RPC params with `bitcoind` as newer versions may be using the rpc cookie route.

Reply:
Fixed.